lib/erb/util.rb

```diff
@@ -5,9 +5,9 @@
 # Rails will not monkey-patch ERB::Escape#html_escape.
 begin
   require 'erb/escape'
-rescue LoadError # JRuby can't load .so
+rescue LoadError # for JRuby, TruffleRuby
```

Comment:
Skipping the require entirely if TruffleRuby or JRuby seems nicer, because that avoids the extra LoadError, which notably causes eager RubyGems loading on TruffleRuby.

$LOAD_PATH.resolve_feature_path might be another way but seems more complicated and might not be available in older Ruby versions.

Reply:
I can make a PR with that as an optimization later, the current PR seems fine behavior-wise

Reply:
pushed https://github.com/ruby/erb/pull/39/commits/769ceb8e3a63ee7d920b807a7db0909e920d70e6. Ruby 3.1-compatible `resolve_feature_path` could optimize away the LoadError now.